joomla/joomla-cms
# [4.0] Handle errors generated by language downloads gracefully

### Summary of Changes
While trying to install a multi-language site I ran into several errors but was not given any information about them on screen. I found them in the XHR request and later in the log file.


### Testing Instructions
1. Get a clean Joomla 4 checkout
2. Start the installation process
3. When the installer is done with the basics, you see this screen:
![image](https://user-images.githubusercontent.com/359377/58745849-f68cfe80-8456-11e9-91d7-5d67d7b96a68.png)
4. Click on Customize Installation
5. Select the `Dutch` language
6. Scroll down and click Next
7. You are back at step 3. If you are lucky you would see a message flash by
8. Apply the patch
9. Select the `Dutch` language
10. Scroll down and click Next
11. Now you stay on the page and see this warning message:
![image](https://user-images.githubusercontent.com/359377/58745871-453a9880-8457-11e9-984f-40db3d118b20.png)

I have another problem with the French language, here you see no warning but an error.
1. Select the `French` language
2. Scroll down and click Next
3. Now you stay on the page and see this error message:
![image](https://user-images.githubusercontent.com/359377/58745906-8e8ae800-8457-11e9-8d5c-412391925e0a.png)


### Expected result
User is shown the error message


### Actual result
Message is shown but page is immediately redirected so impossible to read.


### Documentation Changes Required
None
